crates/aptos/src/common/types.rs

```diff
@@ -294,9 +304,9 @@ pub struct ProfileSummary {
     pub network: Option<Network>,
     pub has_private_key: bool,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub public_key: Option<Ed25519PublicKey>,
+    pub public_key: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub account: Option<AccountAddress>,
+    pub account: Option<String>,
```

Comment:
Same Q: can type safety be preserved?

Reply:
Similarly... this is for a CLI string output